tests_backend/tests/cli/test_cli.ts

```diff
@@ -0,0 +1,203 @@
+import { exists } from 'https://deno.land/std@0.202.0/fs/mod.ts'
+import {
+  mergeReadableStreams,
+} from 'https://deno.land/std@0.201.0/streams/merge_readable_streams.ts'
+import { BlobReader, TextWriter, ZipReader } from 'https://deno.land/x/zipjs/index.js'
+import { type RunnableTest, type SupabaseType, assert } from '../../utils.ts'
+import { getSupabaseSecret, getUpdateBaseData, responseOk, sendUpdate, testPlaywright } from '../../utils.ts'
+
+let cliPath: string | null = null
+let appPath: string | null = null
+const semver = `1.0.${Date.now()}`
+
+// This comes from seed.sql
+const defaultApiKey = 'ae6e7458-c46d-4c00-aa3b-153b0b8520ea'
+const defaultPackageJson = `{
+  "name": "test-cli-app",
+  "version": "1.0.0",
+  "description": "An Amazing Test App",
+  "dependencies": {
+  },
+  "devDependencies": {
+    "@capacitor/cli": "^5.4.1",
+    "typescript": "^5.2.2"
+  },
+  "author": ""
+}`
+const indexJsCode
+= `console.log('Hello world!!!');
+notifyAppReady();\n`
+
+function generateDefaultJsonCliConfig(baseUrl: URL) {
+  return {
+    appId: 'com.demo.app',
+    appName: 'demoApp',
+    webDir: 'dist',
+    bundledWebRuntime: false,
+    plugins: {
+      CapacitorUpdater: {
+        autoUpdate: true,
+        localS3: true,
+        localHost: 'http://localhost:5173',
+        localWebHost: 'http://localhost:5173',
+        localSupa: `http://${baseUrl.host}`,
+        localSupaAnon: getSupabaseSecret(),
+        statsUrl: new URL('stats', baseUrl).toString(),
+        channelUrl: new URL('channel_self', baseUrl).toString(),
+        updateUrl: new URL('updates', baseUrl).toString(),
+      },
+    },
+  }
+}
+
+function generateCliConfig(baseUrl: URL): string {
+  return `import { CapacitorConfig } from '@capacitor/cli';
+
+const config: CapacitorConfig = ${JSON.stringify(generateDefaultJsonCliConfig(baseUrl), null, 2)};
+
+export default config;\n`
+}
+
+export function getTest(): RunnableTest {
+  return {
+    fullName: 'Test cli',
+    testWithRedis: false,
+    tests: [
+      {
+        name: 'Prepare cli',
+        test: prepareCli,
+        execute: 1,
```

Comment:
maybe we could use `skip: false` instead, and the default would be false; what do you think?

set skip to true it would be more clear since execute a test is usually the standard behavior

Reply:
This is more about how many times should the test execute (like the big update endpoint test [here](https://github.com/WcaleNieWolny/capgo/blob/cc80c5866154380533b90395fe547fa96f951041/tests_backend/tests/backend/updates_test.ts#L17))

Reply:
ohhhh, I thought it was a boolean value; in this case, we could use `executions` maybe? `executionsNumber`? IDK, but something that explains that this is the number of executions, `execute` seems like a true/false flag :eyes: